src/pudl/extract/ferc1.py

```diff
@@ -758,10 +758,10 @@ def extract_xbrl(
         return ferc1_raw_dfs
 
     for pudl_table in ferc1_settings.tables:
-        # TODO: Raise exception once XBRL tables are fully integrated
-        # For now skip because map is not defined for all pudl tables
+        if pudl_table not in TABLE_NAME_MAP:
+            raise ValueError(f"{pudl_table} not found in the list of known tables.")
```

Comment:
I went ahead and added these checks for real since we have all the basic pudl tables in now.